lib/githubchart/svg.rb

```diff
@@ -21,6 +22,14 @@ def svg
       chart.to_s
     end
 
+    def svg_square
+      grid = matrix.minor(0..(matrix.row_size-1), (matrix.column_size - 7)..(matrix.column_size - 1))
```

Comment:
We probably want to either cache the matrix in githubchart.rb or cache it here, since this is makeing so many distinct lookups. Or port the sizing option to there